setup.py

```diff
@@ -171,6 +181,113 @@ def get_extensions():
+    else:
+        # Check if using Anaconda to produce wheels
+        conda = distutils.spawn.find_executable('conda')
+        is_conda = conda is not None
+        print('Running build on conda: {0}'.format(is_conda))
+        if is_conda:
+            python_executable = sys.executable
+            if os.name == 'nt':
+                env_folder = os.path.dirname(python_executable)
+                env_library_path = os.path.join(env_folder, 'Library')
+                turbo_include_folder = os.path.join(
+                    env_library_path, 'include')
+                turbojpeg_header = os.path.join(
+                    turbo_include_folder, 'turbojpeg.h')
+                turbo_lib_folder = os.path.join(env_library_path, 'lib')
+                turbojpeg_found = os.path.isfile(turbojpeg_header)
+            else:
+                env_bin_folder = os.path.dirname(python_executable)
+                env_folder = os.path.dirname(env_bin_folder)
+                turbo_lib_folder = os.path.join(env_folder, 'lib')
+                turbo_include_folder = os.path.join(env_folder, 'include')
+                turbojpeg_header = os.path.join(
+                    turbo_include_folder, 'turbojpeg.h')
+                turbojpeg_found = os.path.isfile(turbojpeg_header)
+            conda_installed = turbojpeg_header
+
+    # Try to locate turbojpeg in Linux standard paths
+    if not turbojpeg_found:
+        if sys.platform == 'linux':
+            turbojpeg_found = os.path.exists('/usr/include/turbojpeg.h')
+            turbojpeg_found = turbojpeg_found or os.path.exists(
+                '/usr/local/include/turbojpeg.h')
+        else:
+            # Lookup in TORCHVISION_INCLUDE or in the package file
+            package_path = os.path.join(this_dir, 'torchvision')
+            for folder in vision_include + package_path:
+                candidate_path = os.path.join(folder, 'turbojpeg.h')
+                turbojpeg_found = os.path.exists(candidate_path)
+                if turbojpeg_found:
+                    break
+
+    image_macros += [('JPEG_FOUND', str(int(turbojpeg_found)))]
+    print('turboJPEG found: {0}'.format(turbojpeg_found))
+    if turbojpeg_found:
+        print('Building torchvision with JPEG image support')
+        image_link_flags.append('turbojpeg')
+        if conda_installed:
+            image_library += [turbo_lib_folder]
+            image_include += [turbo_include_folder]
+
+    image_path = os.path.join(extensions_dir, 'cpu', 'image')
+    image_src = glob.glob(os.path.join(image_path, '*.cpp'))
+
+    if png_found or turbojpeg_found:
+        ext_modules.append(extension(
+            'torchvision.image',
+            image_src,
+            include_dirs=include_dirs + [image_path] + image_include,
+            library_dirs=library_dirs + image_library,
+            define_macros=image_macros,
+            libraries=image_link_flags,
+            extra_compile_args=extra_compile_args
+        ))
```

Comment:
Given the complexity of this, could you move it into a separate helper function that returns the `extension`?